### PR TITLE
lib: bin: lwm2m_carrier: update to v0.30.2

### PR DESCRIPTION
**Pull request by nordic-hani**
* library updated to the latest version.

Signed-off-by: Håvard Vermeer <havard.vermeer@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,38 @@ Changelog
...
+
+Certification status
+====================
+
+For certification status, see `Mobile network operator certifications`_.
+
+Changes
+=======
+
+* Added the functions :c:func:`lwm2m_os_lte_modes_get` and :c:func:`lwm2m_os_lte_mode_request`
```
Comment (umapraseeda) [inline doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst:25]:
```suggestion
* Added the functions :c:func:`lwm2m_os_lte_modes_get` and :c:func:`lwm2m_os_lte_mode_request`.
```

**Review by umapraseeda — COMMENTED** [review]
Shouldn't we add an entry under Binary libraries in the changelog (example: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-2.0.0.html#binary-libraries)?

**Comment by eibe85** [comment]
move this out of draft pr and into, dnm?

**Comment by nordic-hani** [comment]
> Shouldn't we add an entry under Binary libraries in the changelog (example: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-2.0.0.html#binary-libraries)?

Done, thanks.

**Review by umapraseeda — APPROVED** [review]

**Review by PFnord — APPROVED** [review]
Verizon - Motive Release testing OK

**Review by stig-bjorlykke — APPROVED** [review]

**Review by nordic-mare — APPROVED** [review]

**Review by lucasvdp — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**